### PR TITLE
Parity

### PR DESCRIPTION
**Pull request by danielakhterov**
- Remove operator requirement from `Client.new()`
    - Implement `Client.fromFile()`
    - Implement `Client.forMainnet()`
    - Implement `Client.forTestnet()`
    - Implement `BaseClient.replaceNodes()`
    - Update `BaseClient.setOperator()` to take AccountId and PrivateKey
    - Implement `BaseClient.setOperatorWith()` which takes AccountId,
      PublicKey and Signer
    - Remove `setShardId`, `setRealmId`, `setTransacitonId` and
    `setNewRealmAccountId` from `*CreateTransaction`
    - Implement `Contract.setContractMemo`
    - Rename `ContractInfo.memo` to `ContractInfo.contractMemo`

### CODE REVIEW

File: src/Status.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
+export enum Status {
+  OK = 0,
```
Comment (mehcode) [inline src/Status.ts]: `Ok`

File: src/index-node.ts [hunk 2]
```diff
@@ -26,6 +50,20 @@ export class Client extends BaseClient {
...
+    }
+
+    public static forTestnet(): Client {
+        return new Client({ network: testnetNodes });
+    }
+
+    public static async fromFile(filename: string): Promise<Client> {
+        const network: Nodes = JSON.parse(await readFile(filename, "utf8"));
+
+        return new Client({ network });
```
Comment (mehcode) [inline src/index-node.ts]: The `fromFile` constructor should support operator as well.

**Comment by mehcode** [comment]
Try an example without operator. I'm pretty sure we're depending on it always being set right now.

**Comment by mehcode** [comment]
@Cooper-Kunz @calvinchengx 

Please review the changes when you get a moment. These should be the last of them before we hit 1.0.

File: examples/create-account.js [hunk 3]
```diff
@@ -32,4 +33,36 @@ async function main() {
     console.log(`account = ${newAccountId}`);
 }
 
-main();
+automaticSign();
```
Comment (mehcode) [inline examples/create-account.js]: the manual sign should be a _new_ example .. perhaps:

 * create-account-with-manual-signing

**Comment by Cooper-Kunz** [comment]
Hey @danielakhterov & @mehcode - at a high level these changes look fine, but I haven't had an opportunity to test them all out. The additional flexibility and customizations within the base client should be nice. Assuming that anything renamed is working towards consistency with the [Java SDK](https://github.com/hashgraph/hedera-sdk-java) - this looks good to me 👍

**Pull request merged**